linebot/event.go

```diff
@@ -216,7 +230,15 @@ func (e *Event) UnmarshalJSON(body []byte) (err error) {
 	case EventTypePostback:
 		e.Postback = rawEvent.Postback
 	case EventTypeBeacon:
-		e.Beacon = rawEvent.Beacon
+		deviceMessage, err := hex.DecodeString(rawEvent.Beacon.DM)
+		if err != nil {
+			return err
```

Comment:
Since it seemed that simple rewriting to `return` got an error `err is shadowed during return`, I divided the variable declaration into separate lines.